src/views/searchResults.js

```diff
@@ -0,0 +1,81 @@
+import icons from '../img/icons.svg';
```

Comment:
use aliases instead manual directory shift.

Reply:
I didn't understand. what do you mean by this?